var/spack/repos/builtin/packages/libgpg-error/package.py

```diff
@@ -32,7 +32,10 @@ class LibgpgError(AutotoolsPackage):
        SmartCard Daemon and possibly more in the future. """
 
     homepage = "https://www.gnupg.org/related_software/libgpg-error"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgpg-error/libgpg-error-1.18.tar.bz2"
+    url      = "https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.26.tar.bz2"
+    list_url = "http://gd.tuwien.ac.at/pub/gnupg/libgpg-error/libgpg-error-1.18.tar.bz2"
+    list_depth = 2
```

Comment:
@healther (and @davydden) Please see the documentation on `list_url` and `list_depth` so that you know what they are used for: http://spack.readthedocs.io/en/latest/packaging_guide.html#list-url